packages/App/src/components/map/Map.tsx

```diff
@@ -79,7 +85,9 @@ const Map: FC = () => {
   }, [allTransactions, chosenMonth]);
 
   const allTransactionExpenses = allTransactionsByMonth.filter(
-    it => it.type === TransactionType.expenses,
+    it =>
+      it.type === TransactionType.expenses &&
+      JSON.stringify(it.coordinate) !== '{}',
```

Comment:
This looks ugly. Maybe you should make `coordinate` nullable and have default value of `null`?